modules/payloads/stagers/java/reverse_http.rb

```diff
@@ -53,8 +53,8 @@ def config
     c << "Spawn=#{spawn}\n"
     c << "URL=http://#{datastore["LHOST"]}"
     c << ":#{datastore["LPORT"]}" if datastore["LPORT"]
-    c << "/"
-    c << generate_uri_checksum(Rex::Payloads::Meterpreter::UriChecksum::URI_CHECKSUM_INITJ, uri_req_len)
+    c << "#{luri}"
+    c << generate_uri_checksum(:init_java, uri_req_len)
```

Comment:
oh, you are trying to do this from msfconsole?
```
use payloads/stagers/java/reverse_http
```
?  That's not a valid thing to do. You can't use a stager directly, it's always in the context of a stage. So, in this case, 'use payload/java/meterpreter/reverse_http' is what you should be testing. That will use the java reverse_http stager.